benchmarks/nnls/datasets/boston.py

```diff
@@ -0,0 +1,24 @@
+from benchopt.base import BaseDataset
```

Comment:
@tomMoral with a growing number of benchmarks, won't it be difficult to maintain several slightly different versions of the same dataset?

Reply:
I agree that we should find better ways to refactor, but this is for another PR.